pkg/domain/entities/engine_image.go

```diff
@@ -40,5 +40,12 @@ type ImageEngine interface { //nolint:interfacebloat
 	ManifestRemoveDigest(ctx context.Context, names, image string) (string, error)
 	ManifestRm(ctx context.Context, names []string) (*ImageRemoveReport, []error)
 	ManifestPush(ctx context.Context, name, destination string, imagePushOpts ImagePushOptions) (string, error)
+	ManifestListClear(ctx context.Context, name string) (string, error)
 	Sign(ctx context.Context, names []string, options SignOptions) (*SignReport, error)
+	FarmName(ctx context.Context) string
+	FarmDriver(ctx context.Context) string
+	FarmStatus(ctx context.Context) error
```

Comment:
I'm not sure this method needs to exist.  Both implementations call their receiver's `Config()` method, throw away the result, and return any error it returned.  None of their callers would be longer if they called `Config()` directly.